### PR TITLE
Adds tofu and garlic recipes to guidebook

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR 
Adds missing Tofu and tofu related recipes. Also adds Garlic Bread.
Quick fixes that I missed during my last PR too.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

Very tiny change.

### CODE REVIEW

File: Resources/Server Info/Guidebook/Chef/pitzah.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 - Cheese wedge
 </Box>
 <Box Orientation="Vertical">
-- 2x raw meat
+- 3x raw meat
```
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Chef/pitzah.xml]: Check the recipe for meat pizza in the actual meal_recipes file. I changed it to 2.

File: Resources/Server Info/Guidebook/Chef/sandvich.xml [hunk 2]
```diff
@@ -55,6 +55,30 @@
...
+<Box Orientation="Vertical">
+- Dough
+- Tofu block
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="20 sec"/>
+<GuideEntityEmbed Entity="FoodBreadTofu" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Garlic Bread Slice[/color]
```
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Chef/sandvich.xml:71]: Making a note here: I will add a full Garlic Bread loaf, so this is more of a placeholder for now.

File: Resources/Server Info/Guidebook/Chef/pitzah.xml (hunk 1 above)
Comment (Torun-S) [reply Resources/Server Info/Guidebook/Chef/pitzah.xml]: Okay, it's actually still on 3, even though I changed that to 2 in my last PR that got merged, and somehow that didn't go through.

So if you could change the meat pizza to 2 meat, that'd be swell.

Comment (FluffiestFloof) [reply Resources/Server Info/Guidebook/Chef/pitzah.xml]: Sure thing

File: Resources/Server Info/Guidebook/Chef/cookbook.xml [hunk 3]
```diff
@@ -147,4 +147,16 @@ Protein can be used raw, but sometimes has to be cooked beforehand. Simply micro
 <GuideEntityEmbed Entity="FoodMeatSalami" Caption=""/>
 </Box>
 
+<Box Orientation="Vertical">
+	[color=#a4885c]Tofu[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 30u soy milk
+- 5u univerzal enzyme
```
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Chef/cookbook.xml]: Denote the enzyme with (c) as a catalyst. If it's not actually a catalyst, make it one.

Comment (FluffiestFloof) [reply Resources/Server Info/Guidebook/Chef/cookbook.xml]: Yep isn't a catalyst, making it be.

**Review by Torun-S — APPROVED** [review]

**Pull request merged**